### DISCUSSION TITLE
Issue with ds.plotsurvfit when dataName is specified

### DISCUSSION THREAD

**Discussion by ghost** [neelsoumya/dsSurvivalClient#13] [Q&A] [answered]
Hello,

When I want to specify `dataName` argument  of `dsSurvivalClient::ds.plotsurvfit` function I obtain the following error : 

" **Error: The 'datasources' were expected to be a list of DSConnection-class objects** "

It seems that it is related to the order of arguments passed into `dsBaseClient::ds.exists` function
 
```
   if (!(is.null(dataName))) {
        defined <- dsBaseClient::ds.exists(datasources, dataName)
    }
```

which is not consistent with the order described in the Help page of the function

![image](https://user-images.githubusercontent.com/123449383/214246359-34695eff-d0e2-4528-be37-425634f19f96.png)

Is it possible to check on your side ?

Thanks

**Comment by StuartWheater** [comment]
I will look at the issue. I suspect that a single DSConnection object is being passed to ds.exists not a list. (as it says in the error message)

**Comment by StuartWheater** [comment]
or just the order

**Comment by neelsoumya** [comment]
Thanks @StuartWheater 

@MatBCHR   can you please share the code that you are using ?

**Comment by StuartWheater** [comment]
Pull request created. Delayed due to setting up testing

**Comment by neelsoumya** [comment]
thanks @StuartWheater 

@MatBCHR  please let us know if you need any help

please use v2.1.2 of dsSurvivalClient

**Comment by neelsoumya** [comment, marked as answer]
now fixed in v2.1.2 of dsSurvivalClient

@StuartWheater @MatBCHR 

Thanks @StuartWheater  for the fix!
